Project.toml

```diff
@@ -19,5 +19,5 @@ CreateParametersExt = "CLIMAParameters"
 CLIMAParameters = "0.8"
 DocStringExtensions = "0.8, 0.9"
 RootSolvers = "0.2, 0.3, 0.4"
-Thermodynamics = "< 0.11.4"
+Thermodynamics = "0.11.5"
```

Comment:
My general preference is to exclude patch numbers here. That's afterall how Julia's compat helper makes recommendations.

Reply:
@Sbozzolo any suggestion?

Reply:
Because of the changes, the new version will not be compatible with Thrmodynamics `0.11.0-0.11.4`, so it should be `0.11.5`.

Reply:
Ah, that's right!